src/DuplicateField.php

```diff
@@ -14,7 +14,7 @@ class DuplicateField extends Field
      */
     public $component = 'duplicate-field';
 
-    public function __construct(string $name, ? string $attribute = null, ? mixed $resolveCallback = null)
+    public function __construct(string $name, ? string $attribute = null, ? $resolveCallback = null)
```

Comment:
Change to : 
`
    public function __construct(string $name, ? string $attribute = null, $resolveCallback = null)

`